frontend/src/app/office-hours/office-hours-page/office-hours-page.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="page-container">
+  <div class="course-heading-row">
```

Comment:
In the future, we might need to wrap this in a card - but in general for this review, I will not comment on strictly UI items (HTML / CSS stuff)! I think that in the summer redesign of the site, changes might happen throughout, so it might be redundant to comment here unless it is vital in the short term. The only thing I may comment on from here on out is where we can use `appearance="outline"`!